wdl-lint/tests/lints/missing-output/source.wdl

```diff
@@ -0,0 +1,5 @@
+version 1.0
+
+workflow test {
+    meta {}
```

Comment:
Does this not trip the missing `parameter_meta` rule also?

Reply:
There's no `input` so the `parameter_meta` check is skipped!

Reply:
Thanks, missed that.